.gitignore

```diff
@@ -7,7 +7,7 @@
 /.env.*.local
 /public/bundles/
 /var/
-/vendor/
+/vendor/p
```

Comment:
You have just comited the folder `vendor` which contains 3rd party modules, and that's 10000 files )) Fix .gitignore file.